docs/website/docs/reference/bindings/index.md

```diff
@@ -32,21 +31,29 @@ See the [C API](./c-api.md) reference page.
 
 See the [Python](./python.md) reference page.
 
-### :simple-javascript: JavaScript
-
-* JavaScript bindings for WebAssembly and WebGPU are under development in IREE's
-[`experimental/web/`](https://github.com/openxla/iree/tree/main/experimental/web)
-directory.
-
-## Unofficial API bindings
+## Unofficial and experimental API bindings
 
 Members of our developer community have authored bindings using other languages:
 
 Language | Compiler API? | Runtime API? | Published packages?
 -------- | ------------ | ----------- | ------------------
+[JavaScript](#javascript) | :grey_question: Experimental | :grey_question: Experimental | :x: Unsupported
+[Java](#java) | :x: Unsupported | :grey_question: Experimental | :x: Unsupported
 [Julia](#julia) | :grey_question: Experimental | :grey_question: Experimental | :x: Unsupported
 [Rust](#rust) | :x: Unsupported | :grey_question: Experimental | :grey_question: Experimental
 
+### :simple-javascript: JavaScript
+
+* JavaScript bindings for WebAssembly and WebGPU are under development in IREE's
+[`experimental/web/`](https://github.com/openxla/iree/tree/main/experimental/web)
+directory.
+
+### :fontawesome-brands-java: Java
+
+* Java TFLite bindings were developed at one point in IREE's
+[`runtime/bindings/tflite/java`](https://github.com/openxla/iree/tree/main/runtime/bindings/tflite/java)
+directory.
```

Comment:
Alternately, we could delete this directory entirely... not sure if anyone has ever used it.

Reply:
probably worth it - I never want to maintain that code :P